tests/functional/RouteTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Functional;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerInterface;
+
+class RouteTest extends TestCase
+{
+    private static ?TestApplicationRunner $application = null;
+
+    public function testRoute()
+    {
+        self::$application = null;
+
+        $method = 'GET';
+        $url = '/';
+
+        $response = $this->doRequest($method, $url);
+
+        $this->assertInstanceOf(ResponseInterface::class, $response);
+
+        $stream = $response->getBody();
+        $stream->rewind();
+        $content = $stream->getContents();
```

Comment:
That's a bit too verbose. Maybe worth adding a shortcut.

Reply:
https://github.com/yiisoft/yii-testing/issues/4